xrpl/wallet/main.py

```diff
@@ -1,44 +1,43 @@
 """The information needed to control an XRPL account."""
-
 from __future__ import annotations
 
-from typing import Type
+from typing import Optional, Type
 
-from xrpl import CryptoAlgorithm
 from xrpl.core.keypairs import derive_classic_address, derive_keypair, generate_seed
 
 
 class Wallet:
     """The information needed to control an XRPL account."""
 
-    def __init__(self: Wallet, seed: str) -> None:
+    def __init__(self: Wallet, pub_key: str, priv_key: str) -> None:
```

Comment:
the reason i did this is to handle a case where a user wants to use a wallet that they did not generate in this session. maybe they don't know the seed? i'll confirm this though

Reply:
The conventional way to create a wallet is from the seed, but it would be reasonable to have a `@classmethod` that derives the wallet from just a private key instead (the `pub_key` is derived from the private key anyway). In that case, the seed would be unknown but that should be fine.